exercises/practice/transpose/.meta/config.json

```diff
@@ -1,10 +1,32 @@
 {
   "blurb": "Take input text and output it transposed.",
-  "authors": [],
+  "authors": [
+    {
+      "github_username": "andrej-makarov-skrt"
+    }
+  ],
+  "contributors": [
+    {
+      "github_username": "SaschaMann"
+    },
+    {
+      "github_username": "kytrinyx"
+    },
+    {
+      "github_username": "Teo-ShaoWei"
+    },
+    {
+      "github_username": "cmcaine"
+    }
+  ],
```

Comment:
This is another deprecated exercise, but it appears to be correct: https://github.com/exercism/julia/commits/v2-final-version/exercises/transpose